monai/networks/layers/convutils.py

```diff
@@ -42,6 +42,34 @@ def same_padding(
     return padding if len(padding) > 1 else padding[0]
 
 
+def nnunet_padding(
```

Comment:
perhaps these two new padding utilities should go to the specific network implementation file, we can address this when you submit a PR about the network impl.

Reply:
I see, I just added necessary blocks for nnunet in this branch, and re-organized the new padding methods in the added place. I'll commit the changes soon.

Reply:
Hi @wyli , I've committed, could you please help to review the updated code. After this branch is merged, I'll open another one and add the rest part (main body) of nnunet.